### ISSUE TITLE
Add CAIRO_* constants

### ISSUE THREAD

**Issue by brocolis** [brocolis/cairo-fortran#1]
Add CAIRO_* constants.

**Comment by vmagnin** [comment]
Hi @brocolis 
the CAIRO enums are in `src/gtkenums-auto.f90`:
```fortran
! cairo-svg.h
enum, bind(c)    !cairo_svg_version_t
  enumerator :: CAIRO_SVG_VERSION_1_1
  enumerator :: CAIRO_SVG_VERSION_1_2
end enum

enum, bind(c)    !cairo_svg_unit_t
  enumerator :: CAIRO_SVG_UNIT_USER = 0
  enumerator :: CAIRO_SVG_UNIT_EM
...
```
Note that this file is not a module, but is included into the `gtk.f90` module:
```fortran
include "gtkenums-auto.f90"
```
You could create a "cairo-enums.f90` module and copy/paste the CAIRO enums there.

**Comment by brocolis** [comment]
@vmagnin, thank you for the fix! I'll merge it.

**Comment by brocolis** [comment]
Fixed in 7676c8848205abcdecc2f4e1a9447dd8b21fcec0. Thank you @vmagnin.

**Comment by vmagnin** [comment]
> Fixed in [7676c88](https://github.com/brocolis/cairo-fortran/commit/7676c8848205abcdecc2f4e1a9447dd8b21fcec0). Thank you @vmagnin.

You could just keep the lines 355-673 where all Cairo constants are, rather than compile 5000 lines which will be for most of them useless for the user.

**Comment by brocolis** [comment]
@vmagnin, I agree and will fix.
